collect_app/src/test/java/org/odk/collect/android/formmanagement/FormMetadataParserTest.java

```diff
@@ -122,6 +122,27 @@ public void cleansUpReferenceManager() throws Exception {
         }
     }
 
+    @Test
+    @SuppressWarnings("PMD.JUnitUseExpected")
```

Comment:
If you agree I think we can suppress this for the whole project. `expected` is pretty useless as it doesn't let you test failure cases (like here) or cases where you need to check the details of an exception. It can also lead to incorrectly passing tests as you're not specifying where the exception should come from, just that the test should throw it at some point.

I didn't realize how much I disliked `expected` until I started writing this 😂